easybuild/easyconfigs/h/HMMER/HMMER-3.0-goalf-1.1.0-no-OFED.eb

```diff
@@ -27,10 +27,8 @@ description = """HMMER is used for searching sequence databases for homologs of
 
 toolchain = {'name': 'goalf', 'version': '1.1.0-no-OFED'}
 
-# eg. http://selab.janelia.org/software/hmmer3/3.0/hmmer-3.0-linux-intel-x86_64.tar.gz
-versiondir = "hmmer%s" % ".".join(version.split('.')[:-1])
-sources = ['hmmer-%s-linux-intel-x86_64.tar.gz' % version]
-source_urls = ['http://selab.janelia.org/software/%s/%s/' % (versiondir, version)]
+source_urls = ['http://eddylab.org/software/hmmer%(version_major)s/%(version)s/']
+sources = ['hmmer-%(version)s-linux-intel-x86_64.tar.gz']
```

Comment:
@fgeorgatos, @verdurin, @rjeschmi, @pescobar: any idea if this is what it's supposed to be?

This is the binary tarball, rather than the source tarball? Cfr. http://hmmer.org/download.html